### DISCUSSION TITLE
TuYa PJ-1203A Smart Meters bombarding the MQTT by Data Packets

### DISCUSSION THREAD

**Discussion by shaik-khizar** [Koenkk/zigbee2mqtt#21705] [Devices]
I have paired TuYa PJ-1203A Smart Meter to zigbee2MQTT. The smart meter is bombarding the MQTT by sending tens of packets of data every second. In expose section its showing update frequency as 10 seconds but the we are receiving lots of packets of  data every second. I tried to reduce the frequency of the data packets received but its not reflecting. Can you tell the solution on how to increase the update frequency so we don't receive data every second.

**Comment by schauveau** [comment]
Indeed. The  PJ-1203A sends multiple zigbee message to update each of the values and each one generates a full MQTT message. So that is something like 15 packets in 1 seconds followed by a pause of 10s. 
 
The solution is to set `debounce: 1` to wait 1 second after the last zigbee message before sending the MQTT message.  See also  https://www.zigbee2mqtt.io/guide/configuration/devices-groups.html#common-device-options

You may also want to have a look at the alternative converter given in https://github.com/Koenkk/zigbee2mqtt/issues/18419#issuecomment-1983353506 
It provides a configuration option to set the delay between the PJ-1203A updates (from 3 to 60s).

**Comment by schauveau** [comment]
Also, I just noticed that `last_seen` (in the global advanced settings) is messing up with `debounce`.  Make sure that it remains disabled. 

See https://github.com/Koenkk/zigbee2mqtt/issues/21953

**Reply by alju67** [reply]
This issue still remains.
Disabling `last_seen` did the trick, but is it possible to disable `last_seen` just for some devices, and not for all in the global settings ?
